nat-lab/tests/helpers.py

```diff
@@ -19,6 +19,21 @@
 
 @dataclass
 class SetupParameters:
+    """
+    Setup parameters for a single Telio node
+
+    # Attributes
```

Comment:
Adding the default value for these would be nice, and maybe taking it a step further than just what is specified in the parameter list. For example, the default value for `derp_servers` as listed in the parameter list is `None`, but if you dig more it's actually `config.DERP_SERVERS`